mona_sdk/client.py

```diff
@@ -483,14 +484,21 @@ def _export_batch_inner(
 
         return client_response
 
-    def _send_mona_rest_api_request(self, messages, default_action=None):
+    def _send_mona_rest_api_request(
+        self, messages, default_action=None, sample_config_name=None
+    ):
         """
         Sends a REST call to Mona's servers with the provided messages.
         :return: A REST response.
         """
-        body = {"userId": self._user_id, "messages": messages}
+        body = {
+            "userId": self._user_id,
+            "messages": messages,
+            "sampleConfigName": sample_config_name,
```

Comment:
this addition to the body is redundant in 99% of the cases, should be conditional only if the sample config name exists...